run/filesystem/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "node index.js",
     "test": "c8 mocha test/index.test.js --exit",
-    "system-test": "c8 mocha test/system.test.js --timeout=180000"
+    "system-test": "NAME=Cloud c8 mocha test/system.test.js --timeout=360000 --exit"
```

Comment:
`NAME=Cloud` not needed. This was setting a env var for another sample

Reply:
removed